samples/examples/income_classifier/train.py

```diff
@@ -1,3 +1,17 @@
+# Copyright 2023 Seldon Technologies Ltd.
```

Comment:
:thought_balloon: We should add a GitHub Action that checks if any (code) files do not include this header, so it has to be present whenever a new file is introduced

Reply:
Agree. Will update any missing for this PR